### ISSUE TITLE
Upgrade to the latest `postcss-values-parser`

### ISSUE THREAD

**Issue by ehmicky** [joscha/node-detective-postcss#16]
The following warning is printed by `npm install node-detective-postcss`:

```
npm WARN deprecated flatten@1.0.2: I wrote this module a very long time ago; you should use something else
```

This is due to `postcss-values-parser` and was fixed by `postcss-values-parser@3.0.0`. Would it be possible to upgrade this dependency? Thanks!

**Comment by joscha** [comment]
@ehmicky can you send a PR please?

**Comment by ehmicky** [comment]
Hi @joscha, thanks for your quick response!

I am not a direct user of `detective-postcss` (I am using `precinct` which uses this) nor am familiar with `postcss-values-parser`. I would need to learn both libraries to be able to send a PR, so I am not sure this might be an option for me at the moment unfortunately :/

**Comment by joscha** [comment]
closed via #107
